src/systems/thruster/Thruster.cc

```diff
@@ -362,10 +392,19 @@ void Thruster::Configure(
 
     feedbackTopic = gz::transport::TopicUtils::AsValidTopic(
         "/model/" + ns + "/joint/" + jointName + "/force");
+
+    this->dataPtr->deadbandTopic = gz::transport::TopicUtils::AsValidTopic(
```

Comment:
check if the output is `empty()` and handle the error

Reply:
same remark as above